src/app/technical-record-create/technical-record-create.component.html

```diff
@@ -1,6 +1,25 @@
 <div class="govuk-width-container create-tech-record">
   <a routerLink="" class="govuk-back-link">Back</a>
   <main class="govuk-main-wrapper">
+
+    <div class="govuk-grid-row" *ngIf="hasErrors">
+      <div class="govuk-grid-column-full">
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+          <h2 class="govuk-error-summary__title govuk-!-font-weight-bold" id="error-summary-title">
```

Comment:
Razvan has added an error summary component. So this might need to changed once his ticket is merged.

Reply:
As we agreed, my PR is going to be merged as it will help unblock 10139 and this issue is going to be solved on Razvan's PR